### PR TITLE
Update math.js

### PR DESCRIPTION
**Pull request by dfarr**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/reviewninja/foo/pull/8" target="_blank">ReviewNinja

**Comment by review-ninja** [comment]
<a href="https://staging.review.ninja/reviewninja/foo/pull/8" target="_blank">ReviewNinja

File: math.js [hunk 1]
```diff
@@ -1,8 +1,4 @@
 module.exports = {
-
-  add: function(a, b) {
```
Comment (dfarr) [inline math.js]: test

File: math.js [hunk 2]
```diff
@@ -14,6 +10,10 @@ module.exports = {
   
   div: function(a, b) {
     return a / b;
+  },
+
```
Comment (dfarr) [inline math.js]: another test

File: math.js [hunk 3]
```diff
@@ -14,6 +10,10 @@ module.exports = {
```
Comment (dfarr) [inline math.js]: one more test

File: math.js (hunk 1 above)
Comment (dfarr) [inline math.js]: there is problem !fix

File: math.js [hunk 4]
```diff
@@ -1,8 +1,4 @@
 module.exports = {
-
-  add: function(a, b) {
-    return a + b;
-  },
   
   sub: function(a, b) {
```
Comment (rwreynard) [inline math.js]: blah blah blah

Comment (dfarr) [inline math.js]: triple blah !fix

**Comment by dtornow** [comment]
MS Pull request comment

File: math.js [hunk 5]
```diff
@@ -14,6 +10,10 @@ module.exports = {
   
   div: function(a, b) {
     return a / b;
+  },
+  
+  multiply: function(a, b) {
+    return a * b;
   }
```
Comment (dtornow) [inline math.js]: MS diff comment

**Pull request merged**